statler_site/templates/play-detail-page.html

```diff
@@ -64,6 +66,11 @@
       <div id="details"></div>
       <div style="height: 200px; border-style: solid;">
         Review post box will go here.
+        <form action="{% url 'api:play-reviews' play_id %}" method="post">
+          {% csrf_token %}
```

Comment:
Also, I'm currently taking a look at json deserialization in Django, so we should have a more realistic picture of how difficult that would be imminently.